boost-common/src/main/java/io/openliberty/boost/utils/ConfigConstants.java

```diff
@@ -17,9 +17,21 @@
     public String HTTP_ENDPOINT = "httpEndpoint";
     public String DEFAULT_HTTP_ENDPOINT = "defaultHttpEndpoint";
 
+    // KeyStore configuration values
+    public String KEYSTORE = "keyStore";
+    public String DEFAULT_KEYSTORE = "defaultKeyStore";
+    public String KEY_ENTRY = "keyEntry"; 
+    public String KEYSTORE_LOCATION = "location";
+    public String KEYSTORE_PASSWORD = "password";
+    public String KEYSTORE_TYPE = "type";
+    public String KEYSTORE_PROVIDER = "provider";
+    public String KEY_NAME = "name";
```

Comment:
I recommend removing KEY_ or KEYSTORE_ from any constant that is fairly generic and could be used on another config element in the future.